tests/cpp/test_sdpa_node.cpp

```diff
@@ -610,4 +634,91 @@ TEST_F(SDPATest, AttnProgram) {
   EXPECT_TRUE(at::allclose(out[0], expected_out));
 }
 
+TEST_F(SDPATest, AttnFwdBwd) {
+  NVFUSER_TEST_CUDA_ARCH_GUARD(8, 0);
+
+  auto fusion = std::make_unique<Fusion>();
```

Comment:
```suggestion
  Fusion fusion;
```
seems enough. `unique_ptr` would be needed for things like FusionExecutorCache, which claims ownership.

Reply:
Once I add scheduling support for bwd pass, I will switch to FusionExecutorCache, so I used unique_ptr.